melos.yaml

```diff
@@ -17,7 +17,7 @@ scripts:
   #  - Requires `flutter_plugin_tools` (`pub global activate flutter_plugin_tools`).
   #  - Requires `git`.
   #  - Requires `clang-format` (can be installed via Brew on MacOS / apt on Ubuntu based).
-  format: pub global run flutter_plugin_tools format --shardIndex 0 --shardCount 3
+  format: dart pub global run flutter_plugin_tools format --shardIndex 0 --shardCount 3
```

Comment:
Using dart implies you have Dart SDK. I believe since it may rely only on Flutter SDK, maybe you can use `flutter format .`